File: Assignment_17/Assignment1.c
#include<stdio.h>

void Pattern(int Row, int iCol)
{
    int i = 0, j = 0;

    for(i = 1; i <= Row; i++)
    {
        for(j = iCol; j>= 1 ;j--)
        {
            if(i==j)
            {
                printf("#\t");
            }
            else
            {
                printf("*\t");
            }
        }

        printf("\n");
    }
}


int main()
{
    
    int iValue1 = 0, iValue2 = 0;

    printf("Enter Number Of Rows :");
    scanf("%d",&iValue1);

    printf("Enter Number Of Colums :");
    scanf("%d",&iValue2);

    Pattern(iValue1,iValue2);


    return 0;
}

File: Assignment____46/Assignment1.java
import java.util.*;

class Pattern
{
    public static void Pattern(int iRow, int iCol)
    {
        for(int i = 1; i <= iRow ; i++)
        {
            char ch = 'A';            
            for(int j = 1 ; j <= iCol; j++)
            {
                System.out.print(ch+"\t");
                ch++;
            }
            System.out.println();
        }
    }
}

class Assignment1
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the Rows :");
        int Row = sobj.nextInt();

        System.out.println("Enter the Colums :");
        int Col = sobj.nextInt();

        Pattern pobj = new Pattern();
        pobj.Pattern(Row, Col);

    }
}